test/Microsoft.ML.Tests/TrainerEstimators/LbfgsTests.cs

```diff
@@ -7,6 +7,9 @@
 using Microsoft.ML.Data;
 using Microsoft.ML.Trainers;
 using Xunit;
+using System.IO;
+using Microsoft.ML.Model;
+using System;
```

Comment:
Not a big deal, just curious what this `.Model` is for.

Reply:
Might go away after rebase. It is in there for IPredictorWithFeatureWeights and ModelFileUtils. 
I think ModelFileUtils got moved. Not sure about IPredictorWithFeatureWeights .

---
In reply to: [265696171](https://github.com/dotnet/machinelearning/pull/2048#discussion_r265696171) [](ancestors = 265696171)